src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala

```diff
@@ -78,6 +78,14 @@ class BitbucketClient(key: String, secretKey: String, token: String, secretToken
     value
   }
 
+  def postForm[T](request: Request[T], values: Map[String, Seq[String]])(implicit reader: Reads[T]): RequestResponse[T] = {
```

Comment:
why would you want to implement an explicit version of post for form and json and make the more general private? the general post is imho the way to go here.